torch/csrc/jit/script/module.h

```diff
@@ -257,11 +257,7 @@ struct TORCH_API Module {
   }
   /// True if the module is in training mode.
   bool is_training() {
-    if (auto p = find_attribute("training")) {
-      return p->value().toBool();
-    }
-    // We are in training mode by default
-    return true;
+    return get_attribute("training").toBool();
```

Comment:
We can't remove this for BC reasons. Existing ScriptModules don't have "training" defined necessarily.